File: 00_조준장/문자열 나누기.py
def solution(s):
    answer = 0
    stack = []

    for i in s:
        stack.append(i)
        if stack.count(stack[0]) == len(stack)/2:
            print(stack)
            stack = []
            answer += 1
    if stack:
        answer += 1

    return answer

File: 02_김성호/문자열나누기.java
package level1;

import java.util.*;


public class PGS_140108 {


    public static int solution(String s) {
        char first = s.charAt(0);
        int count = 0;
        int diff = 0;
        int answer = 0;
      
        for (int i = 0; i < s.length(); i++) {
            if (count == diff) {
                answer++;
                first = s.charAt(i);
            }
            if (first == s.charAt(i)) {
                count++;
            } else {
                diff++;
            }
        }
        return answer;
    }
  
}
